src/packages/examples-ext/processors/exampleDependenciesBuilder.js

```diff
@@ -36,15 +39,25 @@ module.exports = function exampleDependenciesBuilder (readFilesProcessor, log, g
 
       // traverse deployments, check project dependencies and add local file
       // dependencies to the list of generated documents
-      generateExamplesProcessor.deployments.forEach(function(deployment) {
```

Comment:
With my configuration:
```

				deployments: [{
					name: 'default',
					examples: {
						commonFiles: {
							scripts: [
								'https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js',
								'docs/resources/js/examples.js'
							],
							stylesheets: [
								'docs/resources/css/examples.css'
							]
						}
					}
				}],
```

I wind up with an object with numeric keys by the time it gets to that function: `{ '0': { name: 'default', examples: { commonFiles: [Object] } } }`

Which is why i made it compatible with both.